src/OpenTelemetry.ResourceDetectors.AWS/AWSEBSResourceDetector.cs

```diff
@@ -14,7 +14,7 @@ namespace OpenTelemetry.ResourceDetectors.AWS;
 /// <summary>
 /// Resource detector for application running in AWS ElasticBeanstalk environment.
 /// </summary>
-public class AWSEBSResourceDetector : IResourceDetector
+public sealed class AWSEBSResourceDetector : IResourceDetector
```

Comment:
In general, for public classes we shouldn't go from non-sealed to sealed as that would be a breaking change.

Since this is a pre-release package, it's okay to do it here.

Reply:
@srprash @atshaw43 Please review.

Reply:
if proceeding, please add to changelog as well.